torchinfo/layer_info.py

```diff
@@ -121,9 +130,18 @@ def calculate_num_params(self) -> None:
         """
         name = ""
         for name, param in self.module.named_parameters():
-            self.num_params += param.nelement()
+            # Masked models save the parameter with the name "_orig" added
+            # They have a buffer ending with "_mask" which has only 0s and 1s
+            if name[-4:] == "orig":
+                cur_params = int(torch.sum(rgetattr(self.module, name[:-4] + "mask")))
+                # Remove "_orig" for better readability and integration
+                name = name[:-5]
+            else:
+                cur_params = param.nelement()
+
+            self.num_params += cur_params
```

Comment:
Let's move this into a function so we don't need to copy paste the code

Reply:
Resolving as change has been implemented